app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd_1 = command.split()
+    name_of_old_file = cmd_1[1]
+    name_of_new_file = cmd_1[2]
+    if name_of_old_file != name_of_new_file:
+        with open(name_of_old_file, "r") \
+                as file_in, open(name_of_new_file, "w") as file_out:
```

Comment:
use parentheses instead of backslash to wrap the line

Reply:
@Bohdan-Salamakha This Features works only by python 3.10, and i can`t pass flake8 test when use ()